lms/templates/ccx/schedule.underscore

```diff
@@ -12,37 +17,39 @@
     <tbody>
       <% _.each(chapters, function(chapter) { %>
         <tr class="chapter collapsed" data-location="<%= chapter.location %>" data-depth="1">
-          <td class="unit">
-            <a href="#"><i class="fa fa-caret-right toggle-collapse"></i></a>
-            <%= chapter.display_name %>
+          <td class="unit" aria-label="Link chapter <%= chapter.display_name %>">
+            <a href="#" class="toggle-collapse"><i class="fa fa-caret-right"></i></a>
```

Comment:
These should be buttons since they "do something" (toggle state).  Links (```<a>```) are used for controls that go somewhere (whenever you find yourself using ```href="#"``` that is usually the case.)  Change these to buttons and add an ```aria-expanded``` attribute to indicate whether the subsection is expanded ```true``` or collapsed ```false``` Apply to subsections and Units below.